### PR TITLE
Ernestomr87 t40 custom modal

### PR DESCRIPTION
**Pull request by stdevGeekmidget**
Finish	    refs#40 skycoin/skycoin-mobilewallet Create Modal Component
Finish	    refs#31 skycoin/skycoin-mobilewallet On-boarding " Modal Disclaimer Create/Load Wallet"
Add Skycoin and Work_Sans Fonts

### CODE REVIEW

File: src/pages/pincode/pincode.ts [hunk 1]
```diff
@@ -33,41 +40,40 @@ export class PincodePage implements OnInit {
...
-        } else {
-          // error.toString() === 'Error: Device is not secure'
-          this.storageAvailable = false;
-        }
-        this.display = true;
-        loader.dismiss();
-      },
-    );
+    this.nav.setRoot(WalletsPage);
+    // this.generateSeed();
```
Comment (samuelvisscher) [inline src/pages/pincode/pincode.ts]: We should not add commented code

File: src/pages/wallet-detail/wallet-detail.html [hunk 2]
```diff
@@ -3,10 +3,14 @@
     <ion-title mode="ios">My Wallet</ion-title>
   </ion-navbar>
 </ion-header>
-
-<ion-content id="pages-wallet-detail">
+<modal (close)="closeModal()" [header]="'View QR Code'" *ngIf="address" [footer]="'B9UG4KcdGdNRScAt5GxxJvX4J89S5vkRcY'">
```
Comment (samuelvisscher) [inline src/pages/wallet-detail/wallet-detail.html]: Are you sure this is correct?

File: src/theme/variables.scss [hunk 3]
```diff
@@ -3,23 +3,74 @@
...
 // Font path is used to include ionicons,
 // roboto, and noto sans fonts
-$font-path: "../assets/fonts";
+$font-path: "../../assets/fonts";
 
 
 @font-face {
   font-family: 'Montreal';
-  src: url('../assets/custom-fonts/MontrealRegular.ttf') format('truetype');
+  src: url('../../assets/custom-fonts/MontrealRegular.ttf') format('truetype');
```
Comment (samuelvisscher) [inline src/theme/variables.scss]: We can remove montreal

File: src/theme/variables.scss [hunk 4]
```diff
@@ -3,23 +3,74 @@
...
+@font-face {
+  font-family: 'Skycoin';
+  font-style: italic;
+  font-weight: 700;
+  src: url('../assets/skycoin/Skycoin-BoldItalic.woff2') format('woff2'),
+  url('../assets/skycoin/Skycoin-BoldItalic.woff') format('woff');
+}
+
+@font-face {
+  font-family: 'Work Sans';
```
Comment (samuelvisscher) [inline src/theme/variables.scss:52]: I dont think we should be using work sans anywhere?

**Review by samuelvisscher — CHANGES_REQUESTED** [review]

File: src/theme/variables.scss (hunk 4 above)
Comment (stdevGeekmidget) [reply src/theme/variables.scss:52]: image

**Review by samuelvisscher — APPROVED** [review]

**Pull request merged**